Modules/Core/ImageFunction/include/itkLinearInterpolateImageFunction.h

```diff
@@ -133,10 +133,9 @@ class ITK_TEMPLATE_EXPORT LinearInterpolateImageFunction : public InterpolateIma
   EvaluateOptimized(const Dispatch<1> &, const ContinuousIndexType & index) const
   {
     IndexType basei;
-    basei[0] = Math::Floor<IndexValueType>(index[0]);
-    basei[0] = std::max(basei[0], this->m_StartIndex[0]);
+    basei[0] = std::max(Math::Floor<IndexValueType>(index[0]), this->m_StartIndex[0]);
```

Comment:
Of course, replacing three `basei[0]` calls with one _might_ cause a performance improvement. I didn't check, but it certainly wouldn't harm  😃